app/models/friendship.rb

```diff
@@ -1,14 +1,14 @@
 class Friendship < ApplicationRecord
```

Comment:
friendship is not a valid name

Reply:
Please suggest a valid name i think Friendship is appropriate